misk-jdbc/src/main/kotlin/misk/jdbc/DataSourceWrapper.kt

```diff
@@ -0,0 +1,76 @@
+package misk.jdbc
+
+import java.io.PrintWriter
+import java.sql.Connection
+import java.sql.ConnectionBuilder
+import java.sql.ShardingKeyBuilder
+import java.util.concurrent.atomic.AtomicReference
+import java.util.logging.Logger
+import javax.sql.DataSource
+
+internal class DataSourceWrapper(private val simpleName : String?): DataSource {
```

Comment:
Is this similar to HikariDataSource that was previously being used? Can we please link the original source code here so that it is easier to understand?

Reply:
Added a comment, LMK if that is better. If not, what original source code are you referring to?